listen.go

```diff
@@ -94,6 +93,13 @@ type ListenConfig struct {
 	// Default : ""
 	CertClientFile string `json:"cert_client_file"`
 
+	// When the graceful shutdown begins, use this field to set the timeout
+	// duration. If the timeout is reached, OnShutdownError will be called.
+	// The default value is 0, which means the timeout setting is disabled.
+	//
+	// Default: 0
```

Comment:
I think default should be 10s. Having infinite will cause for example docker to send a `SIGKILL`.

Reply:
@efectn @ReneWerner87 Thoughts?

Reply:
@ReneWerner87 Yes, I have changed the default value to 10s in `listenConfigDefault`